options/locale/locale_en-US.ini

```diff
@@ -1628,6 +1628,15 @@ issues.label.filter_sort.alphabetically = Alphabetically
 issues.label.filter_sort.reverse_alphabetically = Reverse alphabetically
 issues.label.filter_sort.by_size = Smallest size
 issues.label.filter_sort.reverse_by_size = Largest size
+issues.development = Development
+issues.maybefixed = May be fixed by %s
+issues.create_branch_from_issue_success = Create branch %s from issue successfully
+issues.base_branch = Base Branch
+issues.pr.completed = Completed
+issues.pr.conflicted = Merge conflicts
+issues.pr.not_exist_issue = Reference issue does not exist.
+issues.branch.latest = Latest commit %s
+issues.link.created = Created %s
```

Comment:
Only one word, no need to add `%s`?

Reply:
Because different languages have different sequences, some would like to put time later but some put front.